### DISCUSSION TITLE
TP3 - Retorno en las funciones

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#97] [Preguntas y respuestas] [answered]
Pregunta para la creacion de funciones es valido usar el void??. Es al hacer la parte del trabajo 1.1 y 1.2 veo que usando los int me devuelven con el ultimo digito que agregue.
Es decir, si yo pongo para cargar: 2 6 8 despues me muestra el arreglo 2688

**Comment by martinvilu** [comment]
Si, pero tené en cuenta que la consigna indica un _posible_ uso para un retorno.
Recordá también que son un tipo de funciones, aplicables también al TP2.

**Reply by luciano-repetti** [reply]
Supongo que en "aplicables también al TP2" quisiste decir "aplicables punto 2" no? Porque si fuese al contrario no entiendo porque algo del tp3 usaríamos en el tp2

**Reply by martinvilu** [reply]
Probablemente

**Comment by martinvilu** [comment]
Podemos dar por cerrado este tema?

**Reply by martinvilu** [reply, marked as answer]
![imagen](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/0d4fcb55-ef71-450d-b377-ebe3acf1827a)

Usen el botón para marcar que esta cerrado, o podemos seguir conversando si no lo está.
